Makefile

```diff
@@ -174,6 +174,11 @@ spellcheck:
 	@echo "==> Spell checking website..."
 	@misspell -error -source=text website/source
 
+apidoc: dev
```

Comment:
On my computer, I can't get this to work unless I add `export VAULT_ADDR=http://localhost:8200` in `gen_openapi.sh` before mounting all the backends. Might need to unset it at the end of the script, too.